### PR TITLE
Fix: Update Variables and Locals in `examples/complete`

### PR DESCRIPTION
**Pull request by max-lobur**
## what
* Update example with current variables

## why
* Example refers deprecated vars

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -122,8 +122,6 @@ module "spacelift" {
   # Cloud Posse recommends pinning every module to a specific version
   # version     = "x.x.x"
 
-  stacks                     = local.stacks
-  stack_config_path          = var.stack_config_path
```
Comment (aknysh) [inline README.md:126]: these are not needed

```
locals {
  config_filenames   = fileset(var.stack_config_path, "*.yaml")
  stack_config_files = [for f in local.config_filenames : f if(replace(f, "globals", "") == f)]
  stacks             = [for f in local.stack_config_files : trimsuffix(basename(f), ".yaml")]
}
```

Comment (max-lobur) [reply README.md:126]: ah, right

File: README.yaml [hunk 2]
```diff
@@ -17,7 +17,7 @@ license: "APACHE2"
 copyrights:
   - name: "Cloud Posse, LLC"
     url: "https://cloudposse.com"
-    year: "2021"
+    year: "2022"
```
Comment (aknysh) [inline README.yaml:20]: no need to change it @max-lobur 
here, it's just the start year

Comment (aknysh) [inline README.yaml:20]:
```suggestion
    year: "2021"
```

Comment (aknysh) [reply README.yaml:20]: otherwise, it will do 

```
Copyright © 2022-2022
```

**Review by nitrocode — APPROVED** [review]

**Review by aknysh — APPROVED** [review]

**Comment by korenyoni** [comment]
> /test all

@max-lobur Terratest isn't set up for this repo (we are missing valid API keys)

**Comment by korenyoni** [comment]
Also please get rid of this variable 

```hcl
± |max-lobur-patch-1 ✓| → tflint --module --enable-rule=terraform_deprecated_index --enable-rule=terraform_unused_declarations --enable-rule=terraform_documented_outputs --enable-rule=terraform_required_providers --enable-rule=terraform_unused_required_providers
1 issue(s) found:

Warning: variable "external_execution" is declared but not used (terraform_unused_declarations)

  on variables.tf line 30:
  30: variable "external_execution" {

Reference: https://github.com/terraform-linters/tflint/blob/v0.27.0/docs/rules/terraform_unused_declarations.md
```

**Review by korenyoni — CHANGES_REQUESTED** [review]
Requested an unused variable to be removed as part of this PR

**Comment by max-lobur** [comment]
@korenyoni https://github.com/cloudposse/terraform-spacelift-cloud-infrastructure-automation/blob/acd61c280a552a4dc8c7858b8338012790ff8661/main.tf#L109

**Comment by korenyoni** [comment]
> @korenyoni
> 
> https://github.com/cloudposse/terraform-spacelift-cloud-infrastructure-automation/blob/acd61c280a552a4dc8c7858b8338012790ff8661/main.tf#L109

Weird, not sure why `tflint` didn't catch that

**Review by korenyoni — APPROVED** [review]
LGTM

**Pull request merged**